types/variables/variables.go

```diff
@@ -1,225 +1,196 @@
 // Copyright 2022 Juan Pablo Tosso and the OWASP Coraza contributors
 // SPDX-License-Identifier: Apache-2.0
 
-//go:generate go run generator/main.go
-
 // Package variables contains the representation of the variables used in the rules
 // Variables are created as bytes and they have a string representation
 package variables
 
+import (
+	"github.com/corazawaf/coraza/v3/internal/variables"
+)
+
 // This file repeats the same content many times in order to make access
 // efficient for seclang and transactions
 
 // RuleVariable is used to identify information from a transaction
-type RuleVariable byte
+type RuleVariable = variables.RuleVariable
```

Comment:
You never stop learning when working with @anuraaga https://yourbasic.org/golang/type-alias/